tensor2tensor/rl/envs/simulated_batch_env.py

```diff
@@ -22,6 +22,7 @@
 from __future__ import print_function
 from tensor2tensor.layers import common_layers
 from tensor2tensor.rl.envs import in_graph_batch_env
+from tensor2tensor.rl.envs import utils
```

Comment:
This is a cyclic import. Please move the needed functions so that the cyclic import is avoided.

Reply:
Done. The cleanest way was to create a new file with batch_env_factory (possibly this should be moved to collect since it is used only there, but at the moment I would leave how it is)